addon/index.js

```diff
@@ -87,6 +87,7 @@ const substr = deprecate(string.substr, 'substr', 'string.substr');
 const substring = deprecate(string.substring, 'substring', 'string.substring');
 const toLower = deprecate(string.toLower, 'toLower', 'string.toLower');
 const toUpper = deprecate(string.toUpper, 'toUpper', 'string.toUpper');
+const trim = deprecate(string.trim, 'trim', 'string.trim');
```

Comment:
Is this correct? Has `trim` existed previously and been deprecated?

Reply:
I have follow `toUpper` behavior. I think it should be better to remove all deprecated stuff `string.*` in one time in order to keep consistency in code.